File: Java&Kotlin lecture/src/main/java/com/java/many/List.java
package com.java.many;

import java.util.ArrayList;
import java.util.Arrays;

public class List {
    public static void main(String[] args) {
        ArrayList list = new ArrayList();
        list.add(2);
        list.add(5);
        list.add(true);
        list.add("ABC");
        System.out.println(list);

        int i = (int)list.get(0);
        String s = (String)list.get(3);

        ArrayList<Integer> list2 = new ArrayList<>();
        list2.add(7);
        list2.add(8);
        System.out.println(list2.size());

        java.util.List<Integer> scores = Arrays.asList(1, 2, 3, 4, 5, 6);
        for(int score: scores) {
            System.out.println(score);
        }
    }
}

File: Java&Kotlin lecture/src/main/kotlin/com/kotlin/many/List.kt
package com.kotlin.many

fun main(args: Array<String>) {
    val list = listOf(0, 1, 2)
    println(list)

    val scores = listOf(0, 1, 2, 3)
    for(score in scores) {
        println(score)
    }

    val mutableList = mutableListOf(5, 1, 2, 7)
    mutableList.add(6)
    println(mutableList)
}